torch/export/_trace.py

```diff
@@ -1414,6 +1377,17 @@ def _strict_export_lower_to_aten_ir(
     # 5. Rename constants nodes in graph module from buffers to constants
     _rename_constants_nodes(gm, export_graph_signature)
 
+    # Prettify names for placeholder nodes.
+    placeholder_naming_pass(
```

Comment:
We do this because there are some param/buffer handling before this line, and this pass relies on the fact that all the param/buffers are settled before calling it.